<launch>
  <arg name="path" value="$(find dr_tracker)"/>
  <arg name="calib_file" default="genius-120"/>
  <arg name="cam_name" default="narrow_stereo"/>

  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video1" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
    <param name="camera_info_url" value="file://$(arg path)/camera_calibration/$(arg calib_file).yaml"/>
    <param name="camera_name" value="narrow_stereo"/>
  </node>
  <node pkg="image_proc" type="image_proc" name="image_proc" ns="usb_cam" output="screen"/>
  <node name="image_view" pkg="image_view" type="image_view" respawn="false">
    <remap from="image" to="/usb_cam/image_rect_color"/>
    <param name="autosize" value="true" />
  </node>
<!--    <node name="image_view" pkg="image_view" type="image_view" respawn="false">-->
<!--    <remap from="image" to="/usb_cam/image_raw"/>-->
<!--    <param name="autosize" value="true" />-->
<!--  </node>-->
</launch>